src/content/self-hosted/administration/configuration.mdx

```diff
@@ -337,12 +337,13 @@ Use `tolerations.devPool` to limit the autoscaler analysis to a subset of cluste
 - `slackWebhook`: A slack webhook url to notify autoscaler events.
 - `volumes.up`: Increase the cluster size when the Volumes in a node vs the max volumes per node is greater than or equal to this value. It defaults to 90 percent.
 - `volumes.down`: Decrease the cluster size when the Volumes in a node vs the max volumes per node is lesser than this value. It defaults to 80 percent.
-
+- `image`: The image to use for the autoscaler. It defaults to `busybox`.
 
 ```yaml
 autoscaler:
   enabled: false
   schedule: 300
+  image: busybox
```

Comment:
I wouldn't add the `image` to the sample. This is for companies that cannot pull from dockerhub